source/use_case_NINA.rst

```diff
@@ -1,118 +1,113 @@
-
-System responses:
-
-
-Alternative Flows
-=================
-
-.. Other, legitimate usage scenarios that can take place within this Use Case.
-
-- The user finds the biodiversity data on a different website, and then only wants to find relavant weather data from MET.
-
-Exceptions
-==========
-
-.. Anticipated error conditions that could occur during execution of the Use Case, and how the
-   system is to respond to those conditions, or the Use Case execution fails for some reason.
-
-Location does not have data, the search is for an area outside our datasets:
-
-- Understandable error message: "Observations not available for this point, closest point with observations is..."
+Producer
+""""""""
 
+User actions:
 
-Includes
-========
+1. Review the literature on Arctic fox distribution and identify plausible (ecologically) variables that could explain the distribution.
+2. Search for occurrence data for Arctic fox in the geographic region of interest could also be survey data (more robust estimates of presence or absence than ad hoc sighting data) 
+   - Note: This search could be carried out using the s-enda catalogue endpoint.
+   - 2.1 Deduplicate the occcurence data, since different sources of data can contain the same data record. 
+   - 2.2 Identify biases in the data generation.
+3. Find covariates that explain the distribution of Arctic fox. Particularly Climate data and observations.
+   - Covariates could be e.g Red fox distribution or locations (from gbif or other); Small mammal records (line transect surveys carried out in Norway and Sweden), altitude (from a DEM), NOA (related to winter conditions), climate data, distance to forest line, etc.
+   - Identify and mitigate for biases in covariates 
+4. Model the distribution of Arctic fox in relation to the covariates taking into account the major bias (which is a lack of absence points) and time. Integrate ancillary information into the model (survival estimates, reproduction rates etc. climate change models) to develop predictive models. 
+5. Model results are made available in the s-enda catalogue.
+   - Check licensing of each of the underlying datasets.
+   - Give appropriate license to newly created dataset. 
 
-N/A
+Consumer
+""""""""
 
-Notes and Issues
-================
+End users of the model could be wildlife managers and NINA researchers.
 
-N/A
+Wildlife management authorities might use the results as a base for their policy decisions. 
+Other scientists might create similar models and compare them, or create a model for a different species and cross reference. 
 
-Relevant software
-=================
 
-N/A
+Alternative Flows
+-----------------
 
-Relevant datasets
-=================
+.. Other, legitimate usage scenarios that can take place within this Use Case.
 
-N/A
+- The occurence data is found on on gbif.org or on data repositories (usually a part of published papers that have archived data) - there are 410 records with location data on gbif. 
```

Comment:
> there are 410 records with location data on gbif

This must be wrong...?

Reply:
taken directly from what was written by mathew here: https://docs.google.com/document/d/1aRldfoX6lVzUc6k0YI8SWjgYVeZToSkunkzLVzWWr-0/edit?usp=sharing

Reply:
This just occurrences for the arctic fox, maybe that is what needs clarification?

Reply:
Ok - we could just delete that sentence as well. It doesn't seem too important(?). Or ask Matthew?

Are we getting anything from @mdlux ? It would be good to close this PR soon.. :)